common/app/views/fragments/containers/facia_cards/videoContainer.scala.html

```diff
@@ -37,9 +42,14 @@ <h2 class="video-title fc-container__header__title">
             @treats(containerDefinition, frontProperties)
         </li>
 
-        @containerDefinition.collectionEssentials.items.filter(i => i.header.isVideo && i.header.hasMainVideoElement.exists(identity)).zipWithIndex.map { case (item, index) =>
+        @containerDefinition.collectionEssentials.items.filter(i => i.header.isVideo).zipWithIndex.map { case (item, index) =>
            <li class="video-playlist__item js-video-playlist-item-@index @if(index == 0){video-playlist__item--active video-playlist__item--first}">
                 @item.properties.maybeContent.map { content =>
+                    @content.atoms.flatMap(_.media.find(_.assets.exists(_.platform == MediaAssetPlatform.Youtube))).map { youTubeAtom =>
```

Comment:
minor: `@content.atoms.flatMap(_.media.find(_.assets.exists(_.platform == MediaAssetPlatform.Youtube)))` 😱  a helper function might be nice here.